### PR TITLE
Get rid of unicode characters from docs

### PR DESCRIPTION
**Pull request by TerryHowe**
These unicode characters are not copy/pasta friendly

### CODE REVIEW

File: docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md [hunk 1]
```diff
@@ -198,8 +198,8 @@ This is likely a [Memory or disk resource problem]({{< relref "#memory-or-disk-r
 ### Generic cluster unavailable
 Troubleshoot more by inspecting bootstrap cluster or workload cluster (depending on the stage of failure) using kubectl commands. 
 ```
-kubectl get pods -A —kubeconfig=<kubeconfig>
-kubectl get nodes -A —kubeconfig=<kubeconfig>
+kubectl get pods -A -kubeconfig=<kubeconfig>
+kubectl get nodes -A -kubeconfig=<kubeconfig>
```
Comment (jaxesn) [inline docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md:202]: Actually doesn't the --kubeconfig need two dashes?

Comment (TerryHowe) [inline docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md:202]:
```suggestion
kubectl get pods -A --kubeconfig=<kubeconfig>
kubectl get nodes -A --kubeconfig=<kubeconfig>
```

Comment (TerryHowe) [reply docs/content/en/docs/tasks/troubleshoot/_troubleshooting.md]: yes fixed

**Pull request merged**